Makefile

```diff
@@ -166,6 +166,11 @@ dev.histogram_decision_timeline_from_kafka:
 	$(call pp,histogram_decision_timeline_from_kafka...)
 	cargo run --bin histogram_decision_timeline_from_kafka --release -- $(args)
 
+## example.replicator_kafka_pg: 🧪 Runs the example replicator with installer for Kafka and Postgres
+example.replicator_kafka_pg:
```

Comment:
GK, I am a little confused. I already have this in line in master branch. Why does it come into master for the second time?

```
make
 Choose a command to run:
  withenv                                      😭 CALL TARGETS LIKE THIS `make withenv RECIPE=dev.init`
  dev.init                                     🌏 Initialize local dev environment
  dev.kafka_init                               🥁 Init kafka topic
  pg.create                                    🥁 Create database
  pg.migrate                                   🥁 Run sql on database
  pg.create_cohort                             🥁 Create database for cohort and applies DB sql
  pg.migrate_cohort                            🥁 Run sql on database for cohort
  install                                      🧹 Installs dependencies
  build                                        🧪 Compiles rust
  dev.run-agent-client                         🧪 Runs agent
  init.samply                                  🧪 Installs Samply profiler
  init.flamegraph                              🧪 Installs Flamegraph profiler
  dev.agent-run-profiler-samply                🧪 Runs agent with Samply profiler
  dev.agent-run-profiler-flamegraph            🧪 Runs agent with Flamegraph profiler
  dev.agent-run-profiler-xcode-cpu             🧪 Runs agent with XCode profiler for Time (CPU) Profiler (OSX only)
  dev.agent-run-profiler-xcode-mem             🧪 Runs agent with XCode profiler for Allocations (OSX only)
  dev.run                                      🧪 Runs rust app in watch mode
  run                                          🧪 Runs rust app
  run.release                                  🧪 Runs rust app in release mode
  run.with_mock_db                             🧪 Runs certifier with mock DB
  dev.preload_db                               🧪 Injects initial data into DB if it's not there yet
  dev.cohort_banking                           🧪 Runs Cohort with built-in replicator and executes banking transactions
  dev.cohort_banking_with_sdk                  🧪 Runs an example of rust app "Cohort Banking" which use cohort_sdk.
  dev.histogram_decision_timeline_from_kafka   🧪 Reads all decisions from kafka and prints processing timeline as csv
  example.replicator_kafka_pg                  🧪 Runs the example replicator with installer for Kafka and Postgres
  dev.run_replicator                           🧪 Runs replicator
  lint                                         🧹 Checks for lint failures on rust
  test.unit                                    🧪 Runs unit tests
  test.some-unit                               🧪 Runs specified unit tests
  test.unit.coverage                           🧪 Runs rust unit tests with coverage 'cobertura' and 'junit' reports
```

Reply:
Aaah, that is because it was not rebased on `master`. I just pushed after rebase so it shouldn't be there anymore